WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -48,7 +49,6 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
             return true
         case .siteIconCreator:
             return BuildConfiguration.current != .appStore
-            return true
```

Comment:
I have removed this as it was raising a warning and would never be executed.

Reply:
Nice catch! That's been there a while too. 😄